src/components/SortByButton/SortByButton.module.css

```diff
@@ -0,0 +1,47 @@
+.sortByButton {
+   width: 25%;
+   height: 4em;
+   font-weight: 300;
+   margin-right: 1em;
+}
```

Comment:
In the future, are you going to give each kind of option-button margin-right just so you have a... ```gap``` between them?